index.js

```diff
@@ -339,7 +348,7 @@ class TuyaDevice extends EventEmitter {
       } catch (error) {
         reject(error);
       }
-    }), this._responseTimeout * 1000, () => {
+    }), this._responseTimeout * 5000, () => {
```

Comment:
Rational on increasing the timeout? I'm ok with bumping it a bit but 5s seems excessive.

Reply:
You're right, I tested it out with 2.5s and it works fine on my side. 1s was just not enough time to get the DP_REFRESH after a SET request and I agree that 5s is excessive. I reduced it in my latest commit :)